### PR TITLE
Fixtures for storage_test

### PR DESCRIPTION
**Pull request by szwedm**
**Description**

storage_test.go in KEB contains functions, which are creating fixed structures required for tests. Some fields of the structures are left with zero values, what could lead to a missed test case. Similar pattern occurs in a lot of other KEB components tests, so it would be a good practice to place fixed structure creation functions in a separate file, where every KEB component test could invoke creational function from one place. This favors code reuse and eliminates redundancy.

Changes proposed in this pull request:

- implement widely used structures in tests as fixtures with fields populated with default values,
- keep fixtures in one source file, which provides default structure creation functions.

**Related issue(s)**
See #428.

### CODE REVIEW

File: components/kyma-environment-broker/internal/storage/storage_test.go [hunk 1]
```diff
@@ -1134,104 +1132,44 @@ func fixInstance(testData instanceData) *internal.Instance {
...
-		},
-		PlatformRegion: "region",
-	}
+	pp := internal.FixProvisioningParameters("test")
+	pp.PlanID = broker.TrialPlanID
+	pp.ServiceID = broker.KymaServiceID
+	return pp
 }
 
 func fixInstanceDetails() internal.InstanceDetails {
```
Comment (polskikiel) [inline components/kyma-environment-broker/internal/storage/storage_test.go:1175]: `fixInstanceDetails` can also be moved to fixture.go

File: components/kyma-environment-broker/internal/fixture.go [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+
+func FixProvisioningParametersDTO() ProvisioningParametersDTO {
+	trialCloudProvider := TrialCloudProvider("provider")
+	return ProvisioningParametersDTO{
+		Name:                        "cluster-name",
+		TargetSecret:                pointer.StringPtr("TargetSecret"),
+		VolumeSizeGb:                ptr.Integer(50),
+		MachineType:                 pointer.StringPtr("MachineType"),
+		Region:                      pointer.StringPtr("Region"),
+		Purpose:                     pointer.StringPtr("Purpose"),
```
Comment (polskikiel) [inline components/kyma-environment-broker/internal/fixture.go:70]: why do we use a new pointer package? you can just use String function from our ptr package

**Review by polskikiel — CHANGES_REQUESTED** [review]
Some comments from my side, do you plan to use common fixtures also in the other KEB's packages? As I suppose it will be added in the next PRs? 
I was also thinking about putting fixtures to a separate package e.g. `fixture/fixture.go` to have it under different name - `fixture.FixDeprovisioningOperation(operationId, testData)` instead of `internal.FixDeprovisioningOperation(operationId, testData)`. Please let me know what do you think.

**Comment by szwedm** [comment]
> Some comments from my side, do you plan to use common fixtures also in the other KEB's packages? As I suppose it will be added in the next PRs?
That's correct, I'm currently replacing fixtures implemented locally in test files with new ones. This will be visible in next PR.
> I was also thinking about putting fixtures to a separate package e.g. `fixture/fixture.go` to have it under different name - `fixture.FixDeprovisioningOperation(operationId, testData)` instead of `internal.FixDeprovisioningOperation(operationId, testData)`. Please let me know what do you think.
Brilliant, I will put fixtures in fixture package.

File: components/kyma-environment-broker/internal/fixture.go (hunk 2 above)
Comment (szwedm) [reply components/kyma-environment-broker/internal/fixture.go:70]: Good point, forgot to change that.

File: components/kyma-environment-broker/internal/storage/storage_test.go (hunk 1 above)
Comment (szwedm) [reply components/kyma-environment-broker/internal/storage/storage_test.go:1175]: Will do.

**Review by polskikiel — DISMISSED** [review]

**Review by polskikiel — APPROVED** [review]

**Pull request merged**